src/viam/app/app_client.py

```diff
@@ -85,7 +85,7 @@
     LocationAuthRequest,
     LocationAuthResponse,
 )
-from viam.proto.app import LogEntry as LogEntryPB
+from viam.proto.common import LogEntry as LogEntryPB
```

Comment:
@njooma sneaking these changes in so things don't break when you eventually merge this PR.